cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1938,6 +1938,17 @@ pub(crate) fn define(
         e.enc_32_64(bound_regmove, rec_frmov.opcodes(vec![0x0f, 0x11]));
     }
 
+    // SIMD integer addition
+    let mut x86_padd_mapping: HashMap<LaneType, Vec<u8>> = HashMap::new();
+    x86_padd_mapping.insert(I8.into(), vec![0x66, 0x0f, 0xfc]); // PADDB from SSE2
+    x86_padd_mapping.insert(I16.into(), vec![0x66, 0x0f, 0xfd]); // PADDW from SSE2
+    x86_padd_mapping.insert(I32.into(), vec![0x66, 0x0f, 0xfe]); // PADDD from SSE2
+    x86_padd_mapping.insert(I64.into(), vec![0x66, 0x0f, 0xd4]); // PADDQ from SSE2
+    for (ty, opcodes) in x86_padd_mapping {
```

Comment:
(First time I realize this, this might apply also in other places). Instead of creating a hashmap, can we do:

```
for &(ty, opcodes) in &[
  (I8, [0x66, 0x0f, 0xfc]),
  (I16, [0x66, 0x0f, 0xfd]) ...
] {
```